bigbluebutton-html5/imports/ui/components/video-dock/component.jsx

```diff
@@ -419,9 +451,11 @@ class VideoDock extends Component {
 
   destroyVideoTag(id) {
     const videos = this.state.videos;
+    const userNames = this.state.userNames;
 
     delete videos[id];
-    this.setState({ videos });
+    delete userNames[id]
```

Comment:
This is wrong. By using `delete` you are actually mutating the current state in memory.
Please do something like this:
```
this.setState({
  userNames: userNames.filter((_,i) => i !== id),
})
```

PS. Fix it for the `videos` too.